pycolmap/ci/vcpkg-dependencies.txt

```diff
@@ -1,19 +0,0 @@
-boost-algorithm
-boost-filesystem
-boost-graph
-boost-heap
-boost-program-options
-boost-property-map
-boost-property-tree
-boost-regex
-boost-system
-ceres[lapack,suitesparse]
-eigen3
-flann
-freeimage
-metis
-gflags
-glog
-gtest
-sqlite3
-jasper[core]
```

Comment:
We need to be very careful that the added vcpkg dependencies do not increase the size of the wheels - i.e. that no additional libraries are linked to COLMAP or its dependencies. So far we have
- excluded `cgal` and `glew`
- explicitly set `jasper[core]` (included by libraw - freeimage) to avoid its `opengl` feature (enabled by default), which would link unnecessary librairies (opengl, gmp, etc.)

Reply:
What's the issue with larger wheels? Is there a maximum binary size enforced by pypi?

Reply:
So, Jasper is needed as a dependency induced from freeimage to handle RAW files? Sorry, not sure I understand that part correctly.

Reply:
freeimage requires libraw (we can't disable this), which requires jasper. Vcpkg installs `jasper[core,default-features,opengl]` but we only need `jasper[core]`. Could we add it to the manifest even though it's not a direct dependency?

Re wheel size: we're very far from the limit of PyPI, it's just more considerate for folks with low bandwidth.

Reply:
Thanks, understood. Re-added jasper[core]. Also added "cgal" as a vcpkg feature.